runners/apex/build.gradle

```diff
@@ -88,6 +88,8 @@ task validatesRunnerBatch(type: Test) {
   testClassesDirs = files(project(":beam-sdks-java-core").sourceSets.test.output.classesDirs)
   useJUnit {
     includeCategories 'org.apache.beam.sdk.testing.ValidatesRunner'
+    // TODO: support @RequiresStableInput then remove the next line
```

Comment:
Having a JIRA link for each runner would be useful. This would give people a place to look to see if it is finished and what the progress is.